CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Mattermost Push Proxy Changelog
 
+
+## 5.8.1 Release
+- Release Date: March 28, 2019
+- Compatible with all versions of Mattermost server and mobile
+
+### Combatibility
```

Comment:
No need for this, as this was part of 5.8.0

Reply:
I would rather have it to be safe